UnitedIncome/components
# Atoms/Illustrations -> TypeScript -> Hooks

### Description
> Please provide a brief description of your changes below.

This is a combination of two projects that demonstrates both TypeScript and React Hooks. In this branch I've migrated all `atoms/illustration` components to be typed, I've also updated the component (and tests) to use React Hooks and the react-testing-library package. I've also used `aria-labelledby` to make these svgs more accessible. 

* Related testing principals: https://testing-library.com/docs/dom-testing-library/api-queries#bytestid

### Testing Instructions
> What should be done in order to test this pull request, or re-create its effects.

* Everything should behave as normal, you should be able to click the illustration and see it toggle correctly.

### Screenshots
> If this pull request includes a visual change please attach any applicable screenshots.

### Comments
> Any additional comments that will aid with the review process.

We're going to be doing a similar presentation on hooks soon, so this is just prep work for that.
